### PR TITLE
feat(tutorial): complete step 1

### PR DESCRIPTION
**Pull request by Ada-siqiliu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,13 @@
...
   },
   "dependencies": {
+    "@carbon/icons-vue": "^10.6.1",
+    "@carbon/vue": "^2.14.0",
+    "carbon-components": "^10.7.0",
     "core-js": "^2.6.5",
     "vue": "^2.6.10",
-    "vue-router": "^3.0.3"
+    "vue-router": "^3.0.3",
+    "yarn": "^1.19.1"
```
Comment (ZrianinaMariia) [inline package.json]: You should not have `yarn` in your dependencies.

File: src/App.vue [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+    <cv-content id="#main-content">
+      <router-view />
+    </cv-content>
   </div>
 </template>
 
-<style lang="scss"></style>
+<script>
+  import {
+    CvButton
```
Comment (ZrianinaMariia) [inline src/App.vue]: You don't need CvButton here anymore since it is not used in this component.

File: src/views/RepoPage/RepoPage.vue [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<template>
+  <div>LANDING PAGE</div>
```
Comment (ZrianinaMariia) [inline src/views/RepoPage/RepoPage.vue]: Should be 'REPO PAGE' here, so that it is easier to check that you router is working as expected.

**Comment by ZrianinaMariia** [comment]
@Ada-siqiliu :cold_sweat: something doesn't look right. Take a look at my comments.

**Comment by Ada-siqiliu** [comment]
Hi,I have modify all problems,please review

**Comment by lee-chase** [comment]
:thumbsup: looks good, you are one step closer to earning the Carbon Developer Essentials Vue badge

**Pull request closed without merging**